llama-index-integrations/llms/llama-index-llms-gemini/llama_index/llms/gemini/base.py

```diff
@@ -145,7 +153,7 @@ def __init__(
             max_tokens = min(max_tokens, self._model_meta.output_token_limit)
 
         super().__init__(
-            model_name=model_name,
+            model=model,
```

Comment:
fyi lets ctrl+f for where `self.model_name` is used (I see at least one other spot, in `def metdata(self)`

Reply:
eh, I fixed this